curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md

```diff
@@ -212,15 +212,22 @@ async (getUserInput) => {
       headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
       body: `description=${expected.description}&duration=${expected.duration}&date=1990-01-01`
     });
-    if (addRes.ok) {
-      const actual = await addRes.json();
-      assert.deepEqual(actual, expected);
-      assert.isString(actual.description);
-      assert.isNumber(actual.duration);
-      assert.isString(actual.date);
-    } else {
+    assert.isTrue(addRes.ok);
+    if (!addRes.ok) {
       throw new Error(`${addRes.status} ${addRes.statusText}`);
     }
+    const responseBody = await addRes.json();
+    const receivedDate = new Date(responseBody.date);
+    const expectedDate = new Date(expected.date); // Jan 1, 1990
+    const allowedPreviousDate = new Date(expectedDate);
+    allowedPreviousDate.setDate(expectedDate.getDate() - 1); // Dec 31, 1989
+    const isValidDate =
+      receivedDate.toDateString() === expectedDate.toDateString() ||
+      receivedDate.toDateString() === allowedPreviousDate.toDateString();
+    assert.isTrue(
+      isValidDate,
+      `Expected date to be ${expectedDate.toDateString()} or ${allowedPreviousDate.toDateString()}, but got ${receivedDate.toDateString()}`
+    );
```

Comment:
You have removed the tests for the username, __id, description and the duration for this section.

Reply:
I have added the tests again.